packages/console/src/pages/ApplicationDetails/index.tsx

```diff
@@ -197,9 +196,7 @@ const ApplicationDetails = () => {
             </div>
           </Card>
           <TabNav>
-            <TabNavItem href={`/applications/${data.id}/settings`}>
-              {t('general.settings_nav')}
-            </TabNavItem>
+            <TabNavItem href={`/applications/${data.id}`}>{t('general.settings_nav')}</TabNavItem>
```

Comment:
We don't have link utils on this page, and we will share all pathnames between all pages in future PRs.